pkgs/tools/security/pinentry/mac.nix

```
@@ -38,10 +38,16 @@ stdenv.mkDerivation rec {
     (allow process-exec (literal "/usr/libexec/PlistBuddy"))
   '';
 
+  strictDeps = true;
   nativeBuildInputs = [ autoreconfHook makeBinaryWrapper texinfo ];
   buildInputs = [ libassuan libgpg-error libiconv Cocoa ];
```

```suggestion
  buildInputs = [ libiconv Cocoa ];
```
can/should we drop them from here then?

---

Good point, I pushed a new commit. I was able to remove the entire buildInputs thanks to the Darwin SDK refactor.